doc/api/crypto.md

```diff
@@ -4217,7 +4217,7 @@ changes:
 
 * `digest` {string} The digest algorithm to use.
 * `ikm` {string|ArrayBuffer|Buffer|TypedArray|DataView|KeyObject} The input
-  keying material. It must be at least one byte in length.
+  keying material. Must be provided but can be zero-length.
```

Comment:
I think it might be a good idea to add such changes to the history in the YAML block above (same for other API changes that are not Web Crypto).